README.md

```diff
@@ -7,6 +7,8 @@
 
 Marquez is an open source **metadata service** for the **collection**, **aggregation**, and **visualization** of a data ecosystem's metadata. It maintains the provenance of how datasets are consumed and produced, provides global visibility into job runtime and frequency of dataset access, centralization of dataset lifecycle management, and much more. Marquez was released and open sourced by [WeWork](https://www.wework.com).
 
+## Quick Demo
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/MarquezProject/marquez/tree/feature/gitpod)
```

Comment:
This needs to be changed to not reference the branch before merging.

Reply:
I'd move this under an "Try it!" section above "Quickstart":

```
## Status
...

## Try it!

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/MarquezProject/marquez/tree/feature/gitpod)

## Quickstart
...
```

Reply:
Sounds good.